winterfell/webapp/src/js/profileService.js

```diff
@@ -34,8 +34,9 @@ app.service('Profile', ['claimService', '$q', 'net', function(claimService, $q,
 
       var that = this;
       net.getUserInfo().then(function success(res) {
-        that.setUser(res.data);
```

Comment:
hmm maybe not anymore. It made sense a while ago when I wanted to test how things flowed.